configuration/local.yml

```diff
@@ -10,12 +10,13 @@ configuration:
     test_name: <any_name>
 
 chrome_flags:
-    third_party_cookie_phaseout: --test-third-party-cookie-phaseout
+    # third_party_cookie_phaseout: --test-third-party-cookie-phaseout
```

Comment:
lets keep the "original" files and create new ones when we need to run with other configs

Reply:
Changes reverted